js/viz/docs/doctreemap.js

```diff
@@ -186,7 +187,19 @@ var dxTreeMap = {
                 color: undefined,
                 opacity: undefined,
                 weight: undefined
-            }
+            },
+            /**
+            * @name dxTreeMapOptions.tile.label.wordWrap
+            * @type Enums.VizWordWrap
+            * @default "normal"
+            */
+            wordWrap: "normal",
+            /**
+            * @name dxTreeMapOptions.tile.label.overflow
```

Comment:
I know that we've decided on this name with @pavelgruba, but now it seems to me that we should prefix it with `text`: `textOverflow`. "Label overflow" sounds a bit weird.